Suite2p/suite2p_main.py

```diff
@@ -0,0 +1,205 @@
+import os
+import sys
+from pathlib import Path
+import matplotlib.pyplot as plt
+import numpy as np
+import h5py
+from typing import Optional
+from tifffile import TiffWriter
+
+sys.path.append("..")
+import utilities as utils
+import definitions as defs
+import suite2p_parameters as s2p_params
+import suite2p_definitions as s2p_defs
+
+import suite2p
+
+# Import for PyInstaller
+from multiprocessing import freeze_support
+freeze_support()
+
+def main(suite2p_params: s2p_params.Suite2pParameters):
+
+    """
+    Suite2p algorithm
+    """
+    filePath: str = suite2p_params.paths[defs.Parameters.Path.FILEPATH]
```

Comment:
Please remember, we name things differently in Python and C++ (snake_case vs camelCase)